CONTRIBUTING.md

```diff
@@ -1,44 +1,26 @@
-![Toolbox logo](data/logo/toolbox-logo-landscape.svg)
+![Toolbox logo](data/gfx/powerup.gif)
```

Comment:
Is @jimmac OK with using this animation here?

Reply:
Didn't ask since the animation is already in the repo. @jimmac?

Reply:
I wouldn't overdo it. Maybe a static illustration for contributions.

Reply:
I don't follow. Is it a problem to use the illustration or is it a problem that it is an animation?

Reply:
Something along this:
![Toolbx_Contributors](https://user-images.githubusercontent.com/14821/144649212-0eb6b44b-d699-4bd8-9f5c-b7cf990111a4.png)

Reply:
Oh, you want to create a new illustration? I mean... sure :). I'd be more than happy to reuse the work you've already done, though.

Btw, lefthanded is good :).